src/Enclosure.php

```diff
@@ -992,7 +992,10 @@ public function embed($options = '', bool $native = false)
         // Flash Media Player file types.
         // Preferred handler for MP3 file types.
         elseif ($handler === 'fmedia' || ($handler === 'mp3' && $mediaplayer !== '')) {
-            $height += 20;
+            if (is_int($height)) {
+                $height += 20;
```

Comment:
What is height is a float? Likely, given that it comes from a [`round()`](https://php.net/round) higher up.
I am afraid several of those changes might hide regressions

Reply:
Looks like that code is already broken. If the default `$width === 'auto'` is used, it will have type [`int|float|'100%'`](https://github.com/simplepie/simplepie/blob/9385c54382edec0a768fdbe310c3e580c1b5dc81/src/Enclosure.php#L948). Later it is being [tested against `'auto'`](https://github.com/simplepie/simplepie/blob/9385c54382edec0a768fdbe310c3e580c1b5dc81/src/Enclosure.php#L956), which the variable cannot hold.

I guess this is another feature to deprecate, just the mention of flash makes me gag.

Reply:
I've gone ahead and adjusted these to use `is_numeric()` instead.